src/main/kotlin/com/terraformation/backend/admin/AdminPlantingSitesController.kt

```diff
@@ -730,6 +732,22 @@ class AdminPlantingSitesController(
     return redirectToAdminHome()
   }
 
+  @PostMapping("/migrateSimplePlantingSites")
+  @RequireGlobalRole([GlobalRole.SuperAdmin])
+  fun migrateSimplePlantingSites(redirectAttributes: RedirectAttributes): String {
+    try {
+      systemUser.run {
+        redirectAttributes.successDetails = plantingSiteStore.migrateSimplePlantingSites()
+      }
+      redirectAttributes.successMessage = "Successfully migrated simple planting sites"
+    } catch (e: Exception) {
+      log.error("Failed to migrate simple planting sites", e)
+      redirectAttributes.failureMessage = "Failed to migrate simple planting sites: ${e.message}"
```

Comment:
Would we want to stay on the migrate page if the migration failed, instead of redirecting to admin home?

Reply:
There isn't a migrate page -- it's a button on the home page.

Reply:
Ah good point, I should have noticed that